schemas/user.json

```diff
@@ -20,7 +22,9 @@
 			"type": "string"
 		},
 		"biography": {
-			"$ref": "#/definitions/LocaleObject"
+			"type": "string",
+			"multilingual": true,
+                        "apiSummary": true
```

Comment:
Indendtation is off.

Reply:
Same for other lines with "apiSummary" below.

Reply:
Fixed (tabs in place of spaces), thanks.